kafka-client-lib/src/main/java/io/jenkins/plugins/remotingkafka/KafkaProducerClient.java

```diff
@@ -28,4 +29,11 @@ public static KafkaProducerClient getInstance() {
         }
         return byteProducer;
     }
+
+    public Producer<String, String> getStringProducer(Properties producerProps) {
+        if (stringProducer == null) {
+            stringProducer = new KafkaProducer<>(producerProps);
+        }
+        return stringProducer;
```

Comment:
There is a risk of creating two producers in the case of race condition. How would it impact the logic?